yarn.lock

```diff
@@ -1639,6 +1639,21 @@
   resolved "https://registry.npmjs.org/@eslint/js/-/js-8.56.0.tgz#ef20350fec605a7f7035a01764731b2de0f3782b"
   integrity sha512-gMsVel9D7f2HLkBma9VbtzZRehRogVRfbr++f06nL2vnCGCNlzOD+/MUov/F4p8myyAHspEhVobgjpX64q5m6A==
 
+"@firebase/component@0.6.8":
```

Comment:
No `@firebase/*` packages should be in yarn.lock. `component` seems to be here because of I think a merge error in packages/data-connect/package.json - the current version of component is 0.6.9. You can run `yarn ts-node scripts/update-internal-dep-versions.ts --file packages/data-connect/package.json` to update any package.json to have the current versions of all internal dependencies.

I'm not sure where `util@1.9.7` is coming from, I can't see that version in any package.json in this branch - although I think it might be a transitive dependency of the older `component` version, so fixing the component version should fix this.